src/clients/rust/Cargo.toml

```diff
@@ -0,0 +1,18 @@
+[package]
+name = "tigerbeetle"
+version = "0.1.0"
+edition = "2021"
+
+[dependencies]
+bitflags = "2.6.0"
+futures = "0.3.31"
+thiserror = "2.0.3"
+
+[build-dependencies]
+anyhow = "1.0.93"
+bindgen = "0.70.1"
```

Comment:
This is a bit unfortunate --- bindgen is a big dependency, and requires quite a bit of compiling on the part of the user! It also arrives at the result in a round-about way: Zig source of truth ->(via tb_client_header.zig) C header -> (via bindgen) Rust code.


What we do for other clients is directly generating code in $LANG from Rust. See how `go_bindings.zig` and `java_bindings.zig` work.